diagnostics-app/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "diagnostics-app",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
```

Comment:
Can you please go through each dependency and double check whether we need all of them at this point?